v1/tester/runner.go

```diff
@@ -320,12 +320,13 @@ func (r *Runner) runTests(ctx context.Context, txn storage.Transaction, enablePr
 		// Activate the bundle(s) to get their info and policies into the store
 		// the actual compiled policies will overwritten later..
 		opts := &bundle.ActivateOpts{
-			Ctx:      ctx,
-			Store:    r.store,
-			Txn:      txn,
-			Compiler: r.compiler,
-			Metrics:  metrics.New(),
-			Bundles:  r.bundles,
+			Ctx:           ctx,
+			Store:         r.store,
+			Txn:           txn,
+			Compiler:      r.compiler,
+			Metrics:       metrics.New(),
+			Bundles:       r.bundles,
+			ParserOptions: ast.ParserOptions{RegoVersion: r.defaultRegoVersion},
```

Comment:
> We should just go over other parts of the package and see if we're not passing the parser options.

@ashutosh-narkar, I found one additional place where we don't pass in parser-options to bundle activation. Since this code-path doesn't expect any existing modules to deactivate, this should be ineffectual; but I put this in the cross-all-t:s-and-dot-all-i:s category.